coc/assets/js/coc.js

```diff
@@ -426,7 +428,7 @@ window.coc = ((window, document, $) => {
               // got merged with lastname on server - output as firstname for compat with old fs
               let uName = obj.firstname;
               let msg = obj.message;
-              let loadedImg = obj.image === '' ? 'https://human-connection.org/wp-content/plugins/coc/assets/images/coc-placeholder.jpg' : obj.image;
+              let loadedImg = obj.image === '' ? cocVars.homeUrl + '/wp-content/plugins/coc/assets/images/coc-placeholder.jpg' : obj.image;
```

Comment:
Prefer string interpolation with backticks and `${variable}`

Reply:
Just for readability, or is there any other reason for that?

Reply:
No, just readability :slightly_smiling_face: